cmd/backup_create.go

```diff
@@ -115,9 +117,9 @@ func backupCreateCommand(l log.Logger, conf config.Optimus, datastoreRepo models
 	return backupCmd
 }
 
-func extractDatastoreName(datastoreRepo models.DatastoreRepo, storerName string) (string, error) {
+func extractDatastoreName(dsRepo models.DatastoreRepo, storerName string) (string, error) {
```

Comment:
If we stick with the current approach, we don't need to pass it as argument (also applicable for the plugin).